src/ServiceControl.Monitoring.Transports.AzureServiceBus/QueueLengthProvider.cs

```diff
@@ -112,6 +127,7 @@ public async Task Stop()
             await managementClient.CloseAsync().ConfigureAwait(false);
         }
 
+        //TODO: agree on the default
```

Comment:
Once per seconds at least I'd say? Should we also switch to a task.whenall to make the queries concurrently? (to make sure that lots of queues doesn't cause delays)